onnxruntime/python/onnxruntime_pybind_state.cc

```diff
@@ -175,6 +177,12 @@ std::string nuphar_settings;
 
 #define PYBIND_UNREFERENCED_PARAMETER(parameter) ((void)(parameter))
 
+// Explicitly provide a definition for the static const var 'GPU' in the OrtDevice struct,
+// GCC 4.x doesn't seem to define this and it breaks the pipelines based on CentOS as it uses
+// GCC 4.x.
+// (This static var is referenced in GetCudaToHostMemCpyFunction())
+const OrtDevice::DeviceType OrtDevice::GPU;
```

Comment:
Breaks CentOS pipelines (uses GCC 4.8.5) without this explicitly provided definition